Disable device tree deprecation warnings, they're burned into ROM and we
cannot do anything about them.

diff -urN linux-5.4.orig/arch/mips/cavium-octeon/executive/cvmx-helper-board.c linux-5.4/arch/mips/cavium-octeon/executive/cvmx-helper-board.c
--- linux-5.4.orig/arch/mips/cavium-octeon/executive/cvmx-helper-board.c	2019-12-22 15:48:45.702104526 +0000
+++ linux-5.4/arch/mips/cavium-octeon/executive/cvmx-helper-board.c	2019-12-22 15:49:50.470135827 +0000
@@ -211,8 +211,10 @@
 {
 	cvmx_helper_link_info_t result;
 
+#if 0
 	WARN(!octeon_is_simulation(),
 	     "Using deprecated link status - please update your DT");
+#endif
 
 	/* Unless we fix it later, all links are defaulted to down */
 	result.u64 = 0;
diff -urN linux-5.4.orig/arch/mips/cavium-octeon/executive/cvmx-helper.c linux-5.4/arch/mips/cavium-octeon/executive/cvmx-helper.c
--- linux-5.4.orig/arch/mips/cavium-octeon/executive/cvmx-helper.c	2019-12-22 15:48:45.702104526 +0000
+++ linux-5.4/arch/mips/cavium-octeon/executive/cvmx-helper.c	2019-12-22 15:50:37.328711499 +0000
@@ -1100,7 +1100,9 @@
 		if (index == 0)
 			result = __cvmx_helper_rgmii_link_get(ipd_port);
 		else {
+#if 0
 			WARN(1, "Using deprecated link status - please update your DT");
+#endif
 			result.s.full_duplex = 1;
 			result.s.link_up = 1;
 			result.s.speed = 1000;
